apps/console/src/features/server-configurations/forms/admin-forced-password-reset.tsx

```diff
@@ -0,0 +1,260 @@
+        = useState<any>(undefined);
+
+    /**
+     * Flattens and resolved form initial values and field metadata.
+     */
+    useEffect(() => {
+
+        if (isEmpty(initialValues?.properties)) {
+            return;
+        }
+
+        const resolvedInitialValues: Map<string, ConnectorPropertyInterface>
+            = new Map<string, ConnectorPropertyInterface>();
+        let resolvedInitialFormValues: any
+            = null;
+
+        initialValues.properties.map((property: ConnectorPropertyInterface) => {
+
+            resolvedInitialValues.set(property.name, property);
+            resolvedInitialFormValues = {
+                ...resolvedInitialFormValues,
+                [ property.name ]: property.value
+            };
+        });
+
+        setInitialConnectorValues(resolvedInitialValues);
+        setInitialFormValues(resolvedInitialFormValues);
+    }, [ initialValues ]);
+
+    /**
+     * Prepare form values for submitting.
+     *
+     * @param values - Form values.
+     * @returns Sanitized form values.
+     */
+    const getUpdatedConfigurations = (values: Record<string, unknown>) => {
+        let data = {};
+
+        for (const key in values) {
+            if (Object.prototype.hasOwnProperty.call(values, key)) {
+                data = {
+                    ...data,
+                    [ GovernanceConnectorUtils.decodeConnectorPropertyName(key) ]: values[ key ]
+                };
+            }
+        }
+
+        return data;
+    };   
+   
+    if (!initialConnectorValues) {
+        return null;
+    }
+
+    return (
+        <Form
+            id={ FORM_ID }
+            uncontrolledForm={ true }
+            initialValues={ initialFormValues }
+            onSubmit={ (values: Record<string, unknown>) => 
+                onSubmit(getUpdatedConfigurations(values)) 
+            }
+        >
+            <Field.Checkbox
+                ariaLabel="Recovery.AdminPasswordReset.RecoveryLink"
+                name={ GovernanceConnectorUtils.encodeConnectorPropertyName(
+                    "Recovery.AdminPasswordReset.RecoveryLink") }
+                className="toggle"
+                label={ GovernanceConnectorUtils.resolveFieldLabel(
+                    "Account Management",
+                    "Recovery.AdminPasswordReset.RecoveryLink", 
+                    "Enable password reset via recovery e-mail") }
+                defaultValue={ initialFormValues?.[ 
+                    "Recovery.AdminPasswordReset.RecoveryLink" ] === "true" }
+                readOnly={ readOnly }
+                disabled={ !isConnectorEnabled }
+                width={ 16 }
+                data-componentid={ `${ testId }-enable-auto-login` }
+                hint={ GovernanceConnectorUtils.resolveFieldHint(
+                    "Account Management",
+                    "Recovery.AdminPasswordReset.RecoveryLink", 
+                    "User gets notified with a link to reset password.")
```

Comment:
Shouldn't we need to i18n these strings?

Reply:
Will provide the i18n placeholder strings in a followup PR